operator/v1/types_ingress.go

```diff
@@ -344,6 +344,52 @@ type IngressControllerSpec struct {
 	// +kubebuilder:default:="Immediate"
 	// +default="Immediate"
 	IdleConnectionTerminationPolicy IngressControllerConnectionTerminationPolicy `json:"idleConnectionTerminationPolicy,omitempty"`
+
+	// closedClientConnectionPolicy maps directly to HAProxy’s "abortonclose" option
+	// and controls how TLS handshakes and request processing are handled when the
+	// client closes the connection.
+	//
+	// Allowed values for this field are "Abort" and "Continue".
+	// The default value is "Continue".
+	//
+	// When set to "Abort", the router will avoid or stop performing the TLS handshake
+	// and will abort any HTTP request that has not yet been sent to the backend
+	// when the client closes the connection.
```

Comment:
What do you mean by "avoid or stop performing"?  Can we just say "stop processing"?
```suggestion
	// When this option is set to "Abort", the router will stop processing
	// the TLS handshake if it is in progress and will not send an HTTP
	// request to the backend server if it has not yet been sent when the
	// client closes the connection.
```